site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java

```diff
@@ -75,11 +75,15 @@ public int compare(Group g1, Group g2){
                 return g1.getTitle().compareToIgnoreCase(g2.getTitle());
         }});
 
-        List<Group> lockedGroupList = site.getGroups().stream().filter(group -> RealmLockMode.ALL.equals(group.getRealmLock()) || RealmLockMode.MODIFY.equals(group.getRealmLock())).collect(Collectors.toList());
-        List<Group> lockedForDeletionGroupList = site.getGroups().stream().filter(group -> RealmLockMode.ALL.equals(group.getRealmLock()) || RealmLockMode.DELETE.equals(group.getRealmLock())).collect(Collectors.toList());
+        // Control the groups that are locked by entities
+        boolean anyGroupLocked = false;
+        List<String> lockedGroupList = new ArrayList<String>();
```

Comment:
can use diamond operator here
ArrayList<>()

Reply:
Well, it's optional right? I like declaring the types but ok, I've changed them.